test/libsolidity/syntaxTests/types/bytes_to_contract.sol

```diff
@@ -0,0 +1,9 @@
+contract C {
+        function f() {
```

Comment:
Can you minify the test a little? The function needs to be public and pure and something like `C(bytes32(0x1234))` would be enough.

Reply:
Done.

Reply:
Visibility and state mutability doesn't seem to be required on syntax tests. Shouldn't that be changed?

Reply:
The analysis stops after the first failing component. This is because the next component might need values computed in the preceding. We can change that in the future, but for now it is like that. Exactly because we might change that in the future, it should be good practice to have as few errors in the code as possible, even if the errors are not displayed by the compiler.

Reply:
Ah sorry, forgot about that.